inviteme/deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Please remove this file. This looks like a helper file for this specific project?